akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -8,7 +8,7 @@ The 2.0 release contains some structural changes that require some
 simple, mechanical source-level changes in client code.
 
 
-Introduced proper named constructor methods insted of ``wrap()``
+Introduced proper named constructor methods instead of ``wrap()``
 ================================================================
```

Comment:
please fix heading marker as well (one `=` is missing now, sphinx is quite picky … )